vnext/local-cli/generator-windows/templates/cs/proj/MyApp.csproj

```diff
@@ -1,184 +1,188 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="16.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="16.0" DefaultTargets="Build" 
+  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
>   [](start = 0, length = 2)

Please remove all the extra new lines in this template file. Especially below when we define properties such as RootNamespace and AssemblyName

Reply:
ah, because these are not valid xml files...

---
In reply to: [428725995](https://github.com/microsoft/react-native-windows/pull/4961#discussion_r428725995) [](ancestors = 428725995)